website/docs/r/compute_instance.html.markdown

```diff
@@ -259,7 +259,7 @@ exported:
 
 * `network_interface.0.address` - The internal ip address of the instance, either manually or dynamically assigned.
 
-* `network_interface.0.access_config.0.assigned_nat_ip` - If the instance has an access config, either the given external ip (in the `nat_ip` field) or the ephemeral (generated) ip (if you didn't provide one).
+* `network_interface.0.access_config.0.nat_ip` - If the instance has an access config, either the given external ip (in the `nat_ip` field) or the ephemeral (generated) ip (if you didn't provide one).
```

Comment:
Revert to `network_interface.0.access_config.0.assigned_nat_ip`

Reply:
Done.